src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  color: #000;
+}
+
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  box-sizing: border-box;
+  width: $field-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $field-pad;
```

Comment:
The variable `$field-pad` is used here but is not defined in this file. Ensure it is imported correctly from `utils/variables` or another file.